src/Microsoft.Maintainability.Analyzers/UnitTests/AvoidUnusedPrivateFieldsTests.Fixer.cs

```diff
@@ -12,22 +14,92 @@ public class AvoidUnusedPrivateFieldsFixerTests : CodeFixTestBase
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpAvoidUnusedPrivateFieldsAnalyzer();
+            return new AvoidUnusedPrivateFieldsAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpAvoidUnusedPrivateFieldsFixer();
+            return new AvoidUnusedPrivateFieldsFixer();
+        }
+
+        [Fact]
+        public void CA1823CSharp()
+        {
+            VerifyCSharpFix(
+                @"  
+class C  
+{  
```

Comment:
Fair enough. I conflated two things:

1. Removing two unused fields at once.
2. Removing an unused field from a line that also declares a used field.

I think we need a test case for No. 2:

```
private int used1, unused, used2;
```

should become

```
private int used1, used2;
```